styles.css

```diff
@@ -1,108 +1,107 @@
-body{
-  max-width: 375px;
-}
-main{
-  width: 100vw;
-  min-height: 100vh;
-  background-image: url("background.png");
-  background-size: 95vw 23vw;
-  background-repeat: no-repeat;
-}
-.navgation {
-  display: flex;
-  flex-wrap: nowrap;
-  flex-grow: 1;
-  max-width: 100vw;
-  max-height: 15vw;
-  align-items: center;
-}
-.menu {
-  max-width: 60vw;
-  max-height: 10vw;
-  margin-left: 72.5vw;
-}
-.para {
-  max-width: 100vw;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 5.5vw;
-  line-height: 6vw;
-  color: #344563;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
-}
-.feyisa {
-  max-width: 100vw;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 14vw; 
-  line-height: 13vw;
-  display: flex;
-  align-items: center;
-  color: #172B4D;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  top: 25%;
-}
-.let {
-  width: 100vw;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 3vw;
-  line-height: 2vw;
-  color: #7F8CFF;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-}
-.part{
-  width: 100vm;
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  
-  }
-.icon{
-display: flex;
-flex-grow: 1;
-max-width: 40vw;
-max-height: 25vm;
-margin-left: 5vm;
-margin-right: 1vm;
-margin-top: 5vm;
-justify-content: space-between;
-}
-.icon-1{
-  display: flex;
-  flex-grow: 1;
-  max-width: 100vw;
-  min-height: 40vh;
-  background-image: url("bottomback.png");
-  background-size: 100vw 20vw;
-  background-repeat: no-repeat;
-  }
-.logo {
-  max-width: 40vw;
-  font-family:'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 5vw;
-  line-height: 5vw;
-  display: flex;
-  align-items: start;
-  text-align: start;
-  color: #6070FF;
-  flex: none;
-  order: 0;
-}
-.bottom{
-  max-width: 5vw;
-  margin-left: 90vw;
-  margin-top: 0vw;
-}
-
+body{
+  max-width: 375px;
+}
+main{
+  width: 100vw;
+  min-height: 100vh;
+  background-image: url("background.png");
+  background-size: 95vw 23vw;
+  background-repeat: no-repeat;
+}
+.navgation {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  max-width: 100vw;
+  max-height: 15vw;
+  align-items: center;
+}

```

Comment:
> - [ ] Good job @feyisakedir👍, Despite making adjustments to the viewport as requested by the code reviewer, there still appears to be a problem with overflow on your page. It is strongly advised that you address this issue, as it negatively affects the user experience.😞
> You can apply `overflow-x: hidden` to the `body` tag and use `%` instead of `vw` for the width.
> 
> Figma design:
> 
> ![overflow2](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/40009354/29b6b0f4-701b-4169-88f8-5cd871ff5d53)
> 
> Your design:
> 
> ![overflow1](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/40009354/7850e81d-a60c-4575-b948-f0892c7fec90)  ![overflow](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/40009354/aab1abf2-5e52-4140-9810-fdb16045144a)
>